### DISCUSSION TITLE
Does it work for Oracle PL/SQL

### DISCUSSION THREAD

**Discussion by rthangam** [community/community#7736] [Copilot Conversations]
Wanted to know if it supports Oracle PL/SQL language ?

**Comment by Urethramancer** [comment]
I don't use any PL/SQL. but it does a fairly good job of recognising SQL in general. It has trouble guessing the specific dialects when mixed with code before you've written any SQL. For instance, I might get MySQL-style prepared statements when writing PostgreSQL API calls in Go.

If writing a whole SQL file with a proper suffix, it usually figures things out when you at least have a line or two of dialect-specific SQL at the start. There's some Oracle SQL to learn from in public repos, so it's likely to do have learned _something_.

**Reply by rthangam** [reply]
Thanks @Urethramancer
